tests/test_warc_to_zim.py

```diff
@@ -453,7 +442,7 @@ def test_args_only(self):
         # error, name required
         with pytest.raises(SystemExit) as e:
             main([])
-            assert e.code == 2
+            assert e.code == 2  # pyright: ignore
```

Comment:
What's being ignored here? magic number ? If yes, it should be in the pyproject for all tests

Reply:
This is fixed now, it should not have been ignored, there was a real issue (do not assert within `pytest.raises`, it won't run).